consensus/src/quorum_store/batch_generator.rs

```diff
@@ -431,7 +431,7 @@ impl BatchGenerator {
                         if back_pressure_increase_latest.elapsed() >= back_pressure_increase_duration {
                             back_pressure_increase_latest = tick_start;
                             dynamic_pull_txn_per_s = std::cmp::min(
-                                dynamic_pull_txn_per_s + self.config.back_pressure.dynamic_min_txn_per_s,
+                                (dynamic_pull_txn_per_s as f64 * self.config.back_pressure.increase_fraction) as u64,
```

Comment:
With multiplicative increase + decrease, I'm a bit concerned the limit will thrash around. Can you kick off a forge run that will hit backpressure so we can see how it behaves?

Reply:
Multiplicative increase + decrease could turn out fine (it's not TCP where you actually lose packets when you over estimate) but I do want to see some experiments and maybe we can tune the factor as well accordingly

Reply:
I'm running some experiments on this [PR](https://github.com/aptos-labs/aptos-core/pull/13601) which additionally uses a more intelligent proof queue that tries to count the remaining transactions in proof queue without duplicates.

In case of pipeline backpressure, we add a delay only as long as there is a backpressure. When there is no backpressure, we don't add any delay in creating a proposal.
In case of QS backpressure, with the above mulitplicative increase, we increase the limit back to the original value 2 seconds after the backpressure reduces. I'm hoping we should be fine.

Reply:
@bchocho I've done the below forge test with the increase fraction and other changes in the PR. The QS backpressure hits here. 

https://github.com/aptos-labs/aptos-core/actions/runs/9559040713